apps/src/templates/studioHomepages/CourseBlocks.jsx

```diff
@@ -208,10 +208,14 @@ export class CourseBlocksHoc extends Component {
 
 export class CourseBlocksIntl extends Component {
   static propTypes = {
-    isTeacher: PropTypes.bool.isRequired,
+    isTeacher: PropTypes.bool,
     showModernElementaryCourses: PropTypes.bool.isRequired,
   };
 
+  static defaultProps = {
+    isTeacher: false,
+  };
```

Comment:
nit: isTeacher will be undefined if it's not passed in, which will evaluate to false in most cases. Could probably delete this, but not worth a Drone re-run